TwitchLib.Api.Core/Common/Helpers.cs

```diff
@@ -26,148 +26,270 @@ public static string AuthScopesToString(AuthScopes scope)
         {
             switch (scope)
```

Comment:
```suggestion
            return scope switch
            {
                     AuthScopes.Chat_Read => "chat:read",
                     AuthScopes.Channel_Moderate => "channel:moderate",
                     ...
                     _ => string.Empty
            };
```

Reply:
Good point. My IDE didn't suggest this, because this whole project is running on C# 7.3 where this feature is not available. I need to push higher version. I will probably update the frameworks in following PR as well to align with Client/Communication projects.